samples/crypto/psa_its/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_LOG=y
+
+CONFIG_NRF_SECURITY=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_MBEDTLS_PSA_CRYPTO_STORAGE_C=y
+
+CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
```

Comment:
I think this must be placed in boards/nrf5340dk_nrf5340_cpuapp_ns.conf, ...
so that it is only included for ns-targets

Reply:
fixed